pyfstat/helper_functions.py

```diff
@@ -1001,24 +1001,9 @@ def predict_fstat(
     cl_pfs = []
     cl_pfs.append("lalapps_PredictFstat")
 
-    if h0 is not None:
-        cl_pfs.append("--h0={}".format(h0))
-    if cosi is not None:
-        cl_pfs.append("--cosi={}".format(cosi))
-    # NOTE: as of lalsuite 6.76, [psi,Alpha,Delta] are required even for
-    # noise-only calls, hence we default them to 0 if h0 is None or ==0,
-    # but fail if they're not set with h0>0.
-    # This can likely be simplified after a future lalsuite release.
-    pars = {"psi": psi, "Alpha": Alpha, "Delta": Delta}
+    pars = {"h0": h0, "cosi": cosi, "psi": psi, "Alpha": Alpha, "Delta": Delta}
     for par in pars.keys():
-        if pars[par] is None:
-            if h0:
-                raise ValueError(
-                    "For h0>0, {:s} needs to be set explicitly.".format(par)
-                )
-            else:
-                cl_pfs.append("--{:s}=0".format(par))
-        else:
+        if pars[par] is not None:
             cl_pfs.append("--{:s}={}".format(par, pars[par]))
```

Comment:
Suggestion:
```
cl_pfs.extend(["--{:s}={}".format(key, value 
for key, value in pars.items() if value is not None])
```

Reply:
Also add a FIXME: The list of `predict_fstat` parameters should be written somewhere, so they can be filtered out from any dictionary without reinventing the loop.

Reply:
There is already `get_predict_fstat_parameters_from_dict()` for that purpose, but its internal list is _not_ identical to the one used for commandline setup here, as there's a `F0`<->`Freq` translation here. We could employ `translate_keys_to_lal`, but I'd like to keep this to a minimum maintenance edit here.

Because, long-term goal: our own PFS class which makes the lalapps call superfluous.